core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -343,6 +353,21 @@ public Counter writeTimeouts() {
         public Counter responseTimeouts() {
             return responseTimeouts;
         }
+
+        @Override
+        public DistributionSummary retryingRequests() {
+            final DistributionSummary retryingRequests = this.retryingRequests;
+            if (retryingRequests != null) {
+                return retryingRequests;
+            }
+
+            final DistributionSummary distributionSummary = parent.summary(idPrefix.name("retrying.requests"), idPrefix.tags());
```

Comment:
Maybe just `"retries"`? (also rename the member field)

Reply:
Thank you. Let me update them.